src/pages/OfferNew/index.tsx

```diff
@@ -84,15 +110,21 @@ export const OfferNew: React.FC = () => {
             />
           )}
         </TrackAction>
-        <Perils insuranceType={getInsuranceType(firstQuote)} />
-        {market === Market.Se && (
-          <Compare currentInsurer={firstQuote.currentInsurer || undefined} />
-        )}
+        {offerData.quotes.map((quote) => {
+          return (
+            <>
+              <Perils contractType={quote.contractType} />
+              {market === Market.Se && (
+                <Compare currentInsurer={quote.currentInsurer || undefined} />
+              )}
+            </>
+          )
+        })}
```

Comment:
When the new Perils are ready I will rearrange this, right now it duplicates the whole thing. Which breaks the design and is thus easily spotted.

Reply:
I'm unsure what you mean? We can get the perils from the contractType but not the quote itself

Reply:
Regardless, it will break at some point when I merge and then I will fix it :D

Reply:
Since I now merged master I no know what you mean, I am fixing so we are stitching in perils on the BundledQuote just as we are doing with the CompletedQuote

Reply:
Fixed this :)